docs/reference/pathformat.rst

```diff
@@ -120,8 +120,10 @@ characters used to surround the disambiguator.
 Any group of albums with identical values for all the identifiers will be
 considered "duplicates". Then, the function tries each disambiguator field,
 looking for one that distinguishes each of the duplicate albums from each
-other. The first such field is used as the result for ``%aunique``. If no field
-suffices, an arbitrary number is used to distinguish the two albums.
+other. The first such field is used as the result for ``%aunique``, preceded
```

Comment:
Since this transitions from the data used for disambiguation to the formatting, would you mind starting a new paragraph here?

In fact, it would be a little more accurate, if we’re going to mention the space here, to also mention the brackets.  That is, the function expands to a space, then the found fields surrounded by brackets (joined by commas). The bracket characters are now even customizable.